modules/ocf/manifests/ssl/setup.pp

```diff
@@ -2,9 +2,14 @@
 # To build and deploy private keys, use the `ocf::ssl::bundle` type.
 # (Or include `ocf::ssl::default` if you just want the default one that includes
 # the FQDN and all aliases of a server.)
+
 class ocf::ssl::setup {
   package { 'ssl-cert':; }
 
+  user { 'ocfletsencrypt':
+    groups => ['ssl-cert', 'sys'],
```

Comment:
I'd add some dependency here to make sure the user exists (to make sure ldap is available) before changing groups here, otherwise I'm worried that puppet will create this user from scratch, and then it'll get shadowed by ldap (or even worse, this user could shadow an ldap user)

Reply:
@jvperrin would making ldap a hard requirement, eg `provider => ldap`, prevent this?

Reply:
Hmmm, it might do, I haven't looked into `provider`s, but that could work

Reply:
I think it may be easiest to just say `forcelocal => false`? the ldap puppet provider appears to require information on connecting to LDAP in puppet.conf.